packages/databyss-api/src/models/Source.js

```diff
@@ -3,51 +3,47 @@ const RichText = require('./RichText')
 
 const Schema = mongoose.Schema
 
-const SourceSchema = new mongoose.Schema(
-  {
-    account: {
-      type: Schema.Types.ObjectId,
-      ref: 'account',
-    },
-    resource: {
-      type: String,
-    },
-    text: RichText,
-    citations: { type: Array, default: [{ textValue: '', ranges: [] }] },
-    // citations: { type: Array, default: [{ textValue: '', ranges: [] }] },
-    authors: {
-      type: Array,
-      default: [{ firstName: { textValue: '' }, lastName: { textValue: '' } }],
-    },
-    date: {
-      type: String,
-    },
-    city: {
-      type: String,
-    },
-    publishingCompany: {
-      type: String,
-    },
-    sourceType: {
+const SourceSchema = new mongoose.Schema({
+  account: {
+    type: Schema.Types.ObjectId,
+    ref: 'account',
+  },
+  resource: {
+    type: String,
+  },
+  text: RichText,
+  citations: { type: Array, default: [{ textValue: '', ranges: [] }] },
+  authors: {
```

Comment:
Let's see if we can declare this as type `[Author]` and have the default be `[Author()]`